Source/Core/DolphinWX/CMakeLists.txt

```diff
@@ -212,6 +212,10 @@ if(wxWidgets_FOUND)
 		install(DIRECTORY ${BUNDLE_PATH} DESTINATION /Applications
 			USE_SOURCE_PERMISSIONS
 			)
+	elseif(WIN32)
+		set_target_properties(${DOLPHIN_EXE} PROPERTIES
+			WIN32_EXECUTABLE ON
```

Comment:
No, it actually maps to this: https://msdn.microsoft.com/en-us/library/fcc1zstk.aspx
Making this a "WINDOWS" application with a WinMain() and not a "CONSOLE" one with a regular main()